.template.config/template.json

```diff
@@ -5,7 +5,7 @@
     "Stacks", 
     "WebApi" 
   ],
-  "name": "Amido Stacks Web Api",
+  "name": "Amido Stacks Web API",
```

Comment:
Perhaps add this same name to the console output

Reply:
Changed console output name